core/builtin.py

```diff
@@ -543,11 +542,10 @@ def Dirs(argv, dir_stack):
     util.warn('*** dirs -l not implemented ***')
   # Following `bash` behavior for order of operations
   if arg.c:
-    while len(dir_stack) > 0:
-      dir_stack.pop()
+    del dir_stack[:]
   elif arg.v:
-    for i in range(0, len(dir_stack)):
-      print(' ' + str(i) + ' ' + dir_stack[i])
+    for i, entry in enumerate(dir_stack):
+      print(' ' + str(i) + ' ' + entry)
```

Comment:
Also this can be `print(' %2d %s' % (i, entry))`

That makes the numbers right justified to 2 chars, which more closely matches what bash does.